mitodl/ocw-hugo-themes
# multiple choice

#### Pre-Flight checklist

<img width="653" alt="Screen Shot 2021-11-30 at 3 04 32 PM" src="https://user-images.githubusercontent.com/1934992/144119978-955a23ed-2614-41c0-9366-bcfa7db14c66.png">
<img width="653" alt="Screen Shot 2021-11-30 at 3 04 58 PM" src="https://user-images.githubusercontent.com/1934992/144119991-1e89fa19-ec92-4f3b-9edb-4871df6e7261.png">
<img width="630" alt="Screen Shot 2021-11-30 at 3 05 16 PM" src="https://user-images.githubusercontent.com/1934992/144119995-1f6fe81d-0418-4901-9a6e-8ca5e1682215.png">

#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/215

#### What's this PR do?
This PR fixes multiple choice questions for legacy courses

#### How should this be manually tested?
Select the branch from https://github.com/mitodl/ocw-to-hugo/pull/418 in ocw-to-hugo

Set example_courses.json in ocw-to-hugo to
```
{
    "courses": [
		"6-0001-introduction-to-computer-science-and-programming-in-python-fall-2016",
		"14-01sc-principles-of-microeconomics-fall-2011",
    ]
}
```
Run
```
node . -i private/input -o private/output -c course_json_examples/example_courses.json --download
```
in ocw-to-hugo.

In ocw-hugo-themes,
Set your .env to
```
OCW_TEST_COURSE=6-0001-introduction-to-computer-science-and-programming-in-python-fall-2016
COURSE_CONTENT_PATH=/ocw-to-hugo/private/output
DOWNLOAD=0
```
Run
```
npm run start:course
```
Go to a page with multiple choice questions and verify that both the "Check" and the "Show Solution" buttons work

Set
OCW_TEST_COURSE=14-01sc-principles-of-microeconomics-fall-2011

Again, go to a page with multiple choice questions and verify that both the "Check" and the "Show Solution" buttons work

Code block rendering and videos are currently broken, that's not related to these changes
